x-pack/plugins/ml/public/application/util/get_selected_ids_url.ts

```diff
@@ -7,10 +7,19 @@ import rison from 'rison-node';
 
 import { getBasePath } from './dependency_cache';
 
-export function getJobIdUrl(tabId: string, jobId: string): string {
-  // Create url for filtering by job id for kibana management table
+export enum TAB_IDS {
+  DATA_FRAME_ANALYTICS = 'data_frame_analytics',
+  ANOMALY_DETECTION = 'jobs',
+}
+
+export function getSelectedIdsUrl(
+  tabId: string,
+  ids: string | string[],
```

Comment:
`getSelectedIdsUrl` is only ever called with a single group Id in an array.
Why does this need to support multiple group ids? should it also support multiple job ids?

Reply:
After testing this I see that we do create a URL containing an array of job IDs, but using the property `jobId`.
IMO this should be a single job id.

Reply:
rather than having a `isGroup` modifier parameter, we could have two functions `getJobIdUrl` and `getGroupIdsUrl` which both call a non-exported common function for creating the url

Reply:
Thanks for taking a look @jgowdyelastic 🙏 
- Support for multiple group ids is requested in the related issue for this as the table also supports it.
- I'd originally put the job Id in the array for consistency and possible support of multiple ids. Taking your feedback into account, I agree it should be just a string until we do support multiple ids (which I don't think we need to anytime soon).

To conclude, groupIds will be an array of ids and jobId will be a string of just one id. I agree that breaking it up into two functions instead of using the `isGroup` parameter would be cleaner. 👍

Reply:
@jgowdyelastic - all suggestions added in 812cd04c507953771404b4a5addf041fcba198a2